src/components/autocomplete/js/autocompleteController.js

```diff
@@ -203,7 +207,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.enableScrolling();
     }
 
-    angular.element($window).off('resize', positionDropdown);
+    angular.element($window).off('resize', onWindowResize);
```

Comment:
I think it's generally a good idea to debounce the resize handlers since they can get called very often.